server/server/src/file_service.rs

```diff
@@ -221,12 +221,15 @@ pub async fn change_doc(
         )?
         .to_lazy();
 
-        let old_usage = get_usage(&tree, db.sizes.data())
+        let old_usage = get_usage_helper(&mut tree, db.sizes.data())
             .unwrap_or_default()
```

Comment:
I'm not sure it makes sense to `unwrap_or_default` here, I think you can `?` and return an internal error in this unexpected case.